using_images/online_images/index.adoc

```diff
@@ -0,0 +1,101 @@
+= Overview
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+This topic group includes information on the different images and accompanying templates that will
+be available as part of the V3 OpenShift Online offering.
+
+The inventory of images are being pulled from a variety of sources, including
+images documented in other sections of link:../index.adoc[Using Images].
+
+The OpenShift Online offering scopes the inventory of images it provides out of the box
+with a few contraints in mind.  Specifically:
+
+* The Project associated with a user can only allocate any number of pods, but
+* A memory limit of 2GiB is place on any project a user creates.  The 2 GiB is spread out across the project's pods, and enforced via the OpenShift Quota system; see the link:../../dev_guide/compute_resources.adoc[computing guide] for a description of how this system works.
+* No images with an emptyDir Kubernets Volume are allowed.
```

Comment:
Couple suggestions for this very tricky section:

- We should state emptyDir volumes are not permitted "at this time". (hopefully they will be soon)
- emptyDir volumes would be in a template rather than an image. (specifically a PodSpec somewhere within a template)
- Specific to images though we should also note that if you are using an image which includes a VOLUME in it's Dockerfile, that volume must be mounted with a persistent volume claim, otherwise the image will not run. (this is also an Online specific limitation)
- We should probably document here that the user will have a quota of 2 persistent volume claims available. (last I heard, the number may change)